docs/doc/dev/doc-generation/description.adoc

```diff
@@ -0,0 +1,45 @@
+= Documentation generation process
+
+== Introduction
+
+The rs-server technical documentation is mainly written using asciidoctor.
+Nevertheless, the python api and the rest api are handled differently.
+The python api is built using sphinx functionalities.
+The rest api is built using fastapi functionalities.
+
+Therefore, the built procedure includes specific steps to handle these elements.
+The technical documentation include links to open the python api and rest api reference guides.
+
+== Technical documentation with multiple pages
+
+link:https://asciidoxy.org/index.html[asciidoxy] is used to generate a asciidoctor documentation with multiple pages and add links between pages.
+asciidoxy is not compatible with the other dev dependencies.
+To use it, we added a python project dedicated to the generation of the technical documentation in the rs-server/docs folder.
```

Comment:
Should be written in a non-personal way, avoid using "We, us, I, they/them". 

E.g
To use it, a python project dedicated to the generation of the technical documentation in the rs-server/docs folder was added.

Reply:
done